### DISCUSSION TITLE
Possibility to save the current value of a bind macro

### DISCUSSION THREAD

**Discussion by NicolasOury** [fonsp/Pluto.jl#1106] [Questions about using Pluto]
Hello,

complete beginner here so apologies if my question is dumb.

I would like to use Pluto for design of boardgame components (deck of cards for example) and I think it would be nice to use bind macro to tweak some parameters (color, font size, position and so on).

However, I would like to also be able to save the tweaks I have done. (For example by having an easy way to set the current value as the default value,with the click of a button).

I imagine it is possible to write a set of macros to sync the tweaked value to a TOML file but I am not sure it is the right approach.

How would you approach this?

Thank you very much for your help!

**Comment by fonsp** [comment]
Currently not yet possible, but that is a great suggestion!
